src/frontend/components/header/header.ts

```diff
@@ -40,12 +40,22 @@ export class Header {
     }
   }
 
+  resetTheme() {
+    this.settingOpened = false;
+    document.removeEventListener('click', this.resetTheme.bind(this), true);
+  }
+
+  openSettings() {
```

Comment:
Since presumably these are not public APIs, is there any reason they couldn't be marked private or protected?